### PR TITLE
planning: modify openspace fallback trajectory 

### PR DESCRIPTION
**Pull request by deidaraho**
open space fallback: modify fallback velocity, acceleration and stopping position.

### CODE REVIEW

File: modules/planning/tasks/deciders/open_space_fallback_decider.cc [hunk 1]
```diff
@@ -54,24 +57,60 @@ Status OpenSpaceFallbackDecider::Process(Frame* frame) {
...
-                              .open_space_fall_back_stop_safety_gap());
-    if (stop_distance > 0.0) {
-      // the accelerate = v0^2 / (2*s), where s is slowing down distance
-      // TODO(Runxin): a better fallback trajectory
+    double relative_collision_point_x = ptr_fallback_trajectory_pair->
+        first[first_collision_idx].path_point().x() -
+        ptr_fallback_trajectory_pair->first[0].path_point().x();
+    double relative_collision_point_y = ptr_fallback_trajectory_pair->
+        first[first_collision_idx].path_point().y() -
+        ptr_fallback_trajectory_pair->first[0].path_point().y();
```
Comment (xiaoxq) [inline modules/planning/tasks/deciders/open_space_fallback_decider.cc]: Save `ptr_fallback_trajectory_pair->first[first_collision_idx].path_point()` and `ptr_fallback_trajectory_pair->first[0].path_point()` to var and reuse.

File: modules/planning/tasks/deciders/open_space_fallback_decider.cc [hunk 2]
```diff
@@ -54,24 +57,60 @@ Status OpenSpaceFallbackDecider::Process(Frame* frame) {
...
+
+      for (size_t i = 1; i < temp_horizon; ++i) {
+        double temp_relative_time =
+            ptr_fallback_trajectory_pair->first[i].relative_time() -
+            previous_point.relative_time();
+        double temp_v = previous_point.v() + accelerate * temp_relative_time;
+
+        if (temp_v <= 0.0) {
+          ptr_fallback_trajectory_pair->first[i].mutable_path_point()->set_x(
+            previous_point.path_point().x());
```
Comment (xiaoxq) [inline modules/planning/tasks/deciders/open_space_fallback_decider.cc]: Indents + 2

**Comment by Capri2014** [comment]
and use const whenever possible

File: modules/planning/tasks/deciders/open_space_fallback_decider.cc (hunk 1 above)
Comment (deidaraho) [reply modules/planning/tasks/deciders/open_space_fallback_decider.cc]: > Save `ptr_fallback_trajectory_pair->first[first_collision_idx].path_point()` and `ptr_fallback_trajectory_pair->first[0].path_point()` to var and reuse.

fixed

File: modules/planning/tasks/deciders/open_space_fallback_decider.cc (hunk 2 above)
Comment (deidaraho) [reply modules/planning/tasks/deciders/open_space_fallback_decider.cc]: > Indents + 2

fixed

**Comment by deidaraho** [comment]
> and use const whenever possible

fixed

**Pull request merged**